packages/react/README.md

```diff
@@ -4,23 +4,23 @@
 
 This package contains helper functionality to make the Confidence SDK work well in a React environment.
```

Comment:
Maybe add a comment that this react package is only relevant if using the Confidence SDK directly and that if using OpenFeature there is an OpenFeature React SDK.